### DISCUSSION TITLE
How to PM change pm_state

### DISCUSSION THREAD

**Discussion by PetrenkoIgor** [zephyrproject-rtos/zephyr#47850] [General]
I use nrf5340 
nrf sdk = 2.0
zepher = lasted
I have a problem with state changes
i use this function  pm_state_force()  but it doesn't give the expected result, what am i doing wrong?
example:
`pm_state_force(0u, &(struct pm_state_info){PM_STATE_SUSPEND_TO_RAM, 1, 0});  // or another pm_states, work only with PM_STATE_SOFT_OFF`

**Comment by carlescufi** [comment]
@gmarull @JordanYates @carlocaione @ceolin

**Comment by JordanYates** [comment]
I don't use system power states myself, but a brief description of what you expect to happen and what is actually happening would be helpful to answer this question.

**Reply by PetrenkoIgor** [reply]
I want to save more energy for more autonomy of my device, any modes would be useful to me except  pm_state_soft_off since it restarts the device from interruption

**Comment by lethang47** [comment]
I also had the same problem. Have you solved it yet?

**Comment by lethang47** [comment]
@carlescufi

**Comment by carlescufi** [comment]
I suggest you open a Nordic Devzone question instead, since you are using NCS.
